config/prow/config.yaml

```diff
@@ -540,8 +540,12 @@ branch-protection:
               protect: true
             release-1.26:
               protect: true
+            release-1.27:
+              protect: true
             kubetest2-aks:
               protect: true
+            gh-pages:
+              protect: false
```

Comment:
This change is not outlined/described in the PR body.

Reply:
@cblecker it is listed in description. Thanks!